ViewModel/HyvaGridViewModel.php

```diff
@@ -45,7 +47,7 @@ class HyvaGridViewModel implements HyvaGridInterface
 
     private string $gridName;
 
-    private array $memoizedColumnDefinitions;
+    private array $memorizedColumnDefinitions;
```

Comment:
Sorry, the correction is wrong - please change it back to `memoizedColumnDefinitions`. :)